test/e2e/specs/editor/various/pattern-overrides.spec.js

```diff
@@ -148,6 +148,8 @@ test.describe( 'Pattern Overrides', () => {
 			const paragraphs = patternBlocks.first().getByRole( 'document', {
 				name: 'Block: Paragraph',
 			} );
+			// Ensure the first pattern is selected.
+			await patternBlocks.first().selectText();
```

Comment:
Why do we need this?

Reply:
It seems that if I don't select the pattern, the blocks always have `inert="true"`.

Reply:
Hmm, interesting! I guess I just didn't notice it before 😅.